roles/lib_openshift/src/class/oc_route.py

```diff
@@ -55,7 +55,13 @@ def create(self):
 
     def update(self):
         '''update the object'''
-        # need to update the tls information and the service name
+        # when the host attribute is being updated, we need to delete and recreate
```

Comment:
It doesn't seem like this should be necessary, I do not see anything in the api docs or api code to indicate that host is immutable.